site/content/en/docs/Concepts/projects.md

```diff
@@ -92,6 +93,21 @@ data:
     readOnly: true
 ```
 
+You can use S3 prefixes to be more restrictive about access to data. For example, if you want to allow access to the 
+`foo` folder or `my-bucket` and it's sub-folders you would declare the location as:
```

Comment:
Should this be "of" not "or"? 

"...access to the `foo` folder **_of_** `my-bucket`..."?

Reply:
Good find, fixed in latest commit